everywherejacobkim/project-bc-spca-renewal
# Developed cards section

## Purpose

Developed basic layout for cards section

## Steps to check/test this feature/bugfix

- Nothing to check, it's processing. 
- I need to redesigned the code to create reusable components

## Screenshots / Videos

-
